enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -0,0 +1,340 @@
+---
+title: tokenized-auth-enablement-operators-on-cloud
+authors:
+  - "@bentito"
+  - "@jharrington22"
+  - "@gallettilance"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@joelanford" # for OLM, OperatorHub changes
+  - "@bparees" # for OLM, OperatorHub changes
+  - "@csrwng" # for HyperShift changes
+  - "@spadgett" # for Console changes
+  - "@abutcher" # for CCO changes
+approvers:
+  - "@sdodson"
+  - "@joelanford"
+  - "@bparees"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@JoelSpeed " # for CredentialsRequest changes
+creation-date: 2023-01-21
+last-updated: 2023-04-19
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OCPBU-4
+  - https://issues.redhat.com/browse/PORTENABLE-471
+  - https://issues.redhat.com/browse/CCO-286
+see-also:
+  - []
+replaces:
+  - []
+superseded-by:
+  - []
+---
+
+# Tokenized Authentication for Red Hat Operators on Cloud Providers
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Many Cloud Providers offer services that allow authentication via Temporary Access Tokens (TAT).
+
+For example, Amazon Web Services (AWS) Secure Token Service (STS) allows you to request temporary limited-privilege
+credentials. A concrete example is write permission on an S3 bucket on AWS. STS is supported on all the major cloud
```

Comment:
Might be nice to enumerate the others here, and perhaps explain why TAT is useful / worth an investment.

Reply:
You mean versus long lived credentials? This has essentially become best practices across the hyperscalers over the past 4 years.

Reply:
Sure - just pointing out that at least to fresh eyes the "why" - even if obvious - is missing from the motivations in the doc :)

Reply:
I'm going to mark this resolved. The cloud providers do have other, older, authentication methods, but the motivation for this enhancement is based on their offering STS and other methods that provide short-lived creds. That's covered.